Gemfile.lock

```diff
@@ -126,7 +126,7 @@ GEM
       railties (>= 5.0.0)
     faker (2.14.0)
       i18n (>= 1.6, < 2)
-    faraday (0.17.3)
```

Comment:
This is used by our oauth2 client lib. I guess we should make sure integrations with Nomis are still working on staging.